api_server/src/main/java/io/zetch/app/domain/User.java

```diff
@@ -39,4 +52,13 @@ public String getUsername() {
     public void setUsername(String username) {
         this.username = username;
     }
+
+    @Override
+    public String toString() {
```

Comment:
Are the double quotes for compatibility issues? Otherwise, I suggest to simplify this to
```java
return String.format(
  "User{username=%s, name=%s, email=%s}",
  username, 
  name,
  email
);
```